### PR TITLE
CMake: install demo version installation scripts into ${CMAKE_INSTALL_DOCDIR}/demo

### PR DESCRIPTION
**Pull request by oleg-derevenetz**
discussed in #3669

Hi @winterheart how about this variant?

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -46,5 +46,16 @@ endif(ENABLE_ZLIB)
...
+)
+install(
+    FILES script/demo/demo_unix.sh
+    PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_EXECUTE GROUP_READ WORLD_EXECUTE WORLD_READ
+    DESTINATION ${CMAKE_INSTALL_DOCDIR}/demo
+)
+install(
+    FILES script/demo/demo_windows.bat script/demo/demo_windows.ps1
+    DESTINATION ${CMAKE_INSTALL_DOCDIR}/demo
+)
```
Comment (ihhub) [inline CMakeLists.txt:61]: How does it work for different systems? I see no indication about OS type.

Comment (oleg-derevenetz) [reply CMakeLists.txt:61]: Hi @ihhub Well, currently it just installs all scripts into `doc/scripts`, no per-OS logic. User should choose which one to run, but package for particular OS may help in this.

Comment (oleg-derevenetz) [reply CMakeLists.txt:61]: May be @winterheart may suggest how to do this per-OS in a portable way.

Comment (ihhub) [reply CMakeLists.txt:61]: How about simple check for Windows and the rest like [here](https://stackoverflow.com/a/9742290)?

Comment (oleg-derevenetz) [reply CMakeLists.txt]: Yes, it seems that this should work.

**Comment by ihhub** [comment]
@oleg-derevenetz , thanks a lot for this improvement!

**Pull request merged**